### PR TITLE
added en-st baseline for jw300 test set and autshumato evaluation set

### PR DESCRIPTION
**Pull request by jasonrobwebster**
I've added baseline BLEU scores for the
- JW300 test set
- Autshumato Evaluation set

In both cases, the model was trained on JW300 training data.

The original Masakhane notebook was split into processing and training notebook in order to run them on seperate Google Collab instances (as the processing step managed to consume the time allocation for a Collab instance).

Custom code was added to the training and testing notebook in order to calculate the BLEU score using multiple human translation references provided by the Autshumato evaluation set. The BLEU score calculation for the JW300 testing set was unaltered.

Results for each set are detailed in the `README.md` file present in each folder.

### CODE REVIEW

**Comment by juliakreutzer** [comment]
Awesome! :sunglasses: 
Can you please add the results to `languae_pairs.md` as well?

**Comment by jasonrobwebster** [comment]
Done!
